wom/src/main/scala/wdl/util/JsUtil.scala

```diff
@@ -80,6 +80,7 @@ object JsUtil {
       case WdlFloat(double) => double.asInstanceOf[java.lang.Double]
       case WdlBoolean(boolean) => boolean.asInstanceOf[java.lang.Boolean]
       case WdlArray(_, array) => array.map(toJavascript).toArray
+      case WdlSingleFile(path) => path
```

Comment:
Huh, I'm curious where exactly this is being used (and why we never stuck problems with no "File to javascript" conversion in the past)

Reply:
And if this is being used somewhere important, can we add `WdlPair` and non-empty `WdlOptional`s in too?

Reply:
This code should probably be in the CWL package.

It is being used to convert inputs to javascript data.  I don't think we've ever done this in the past, but I think you know this so I must be misunderstanding your point...

CWL don't support pairs nor optionals.

Reply:
Oh right, I didn't realize this was a CWL specific thing (which explains why we never hit it before)! 

I asked about non-empty optionals because there actually is a handler for empty optionals in this chain